hnn_core/gui/gui.py

```diff
@@ -490,10 +265,15 @@ def __init__(self, theme_color="#8A2BE2", log_window_height="100px",
         # Connectivity list
         self.connectivity_widgets = list()
 
-        # In-memory storage of all simulation related simulation_data
-        self.simulation_data = defaultdict(lambda: dict(net=None, dpls=list()))
-
         self._init_ui_components()
+        self.add_logging_window_logger()
+
+    def add_logging_window_logger(self):
+        handler = _OutputWidgetHandler(self._log_out)
+        handler.setFormatter(
+            logging.Formatter('%(asctime)s  - [%(levelname)s] %(message)s'))
+        logger.addHandler(handler)
+        logger.setLevel(logging.DEBUG)
```

Comment:
don't forget to set this to INFO !

Reply:
I'm still debugging. I'll remove it before setting the status to MRG

Reply:
not to be forgotten!